### DISCUSSION TITLE
New Sensor Start Failure

### DISCUSSION THREAD

**Discussion by GLoren1** [NightscoutFoundation/xDrip#1916] [General]
I changed my sensor. this morning.  xDrip accepted the code and started the sensor; however a couple of minutes later it stopped.

The logs show:
Session start failed 1969-16-31 18:00:00

I've tried to restart 3 times with the same error.

**Comment by Navid200** [comment]
When you say you tried to restart three times, do you mean you removed the transmitter every time?
These are the guidelines for a restart:
https://github.com/Navid200/xDrip/wiki/Restart-G6-Sensor

**Reply by GLoren1** [reply]
I read the guidelines and removed the sensor for about 30 minutes.  There was no operation in queue, so I started the sensor with the code. After a minute or so, it stopped again with the same error codes as before.

**Reply by Navid200** [reply]
You say you removed the sensor.
Do you mean you removed the transmitter?

**Reply by GLoren1** [reply]
Yes, I removed the transmitter--not sensor

**Reply by Navid200** [reply]
Can you post a screenshot of the system status page?
If you blackout the serial number, please leave the first two characters visible.
I suspect the number of days is greater than 99.

**Reply by GLoren1** [reply]
Yes it is greater than 100 days.  Can this transmitter be reset?
![temp (2)](https://user-images.githubusercontent.com/62320462/148664031-f4192886-e5b2-4184-8d05-f7e5b7b00b7b.jpg)

**Reply by Navid200** [reply]
Unfortunately, no:
https://github.com/Navid200/xDrip/wiki/Transmitter-Lifetime

**Reply by GLoren1** [reply]
Thank you.  I have a new transmitter that expired 12/2021. Should that work if I use it now?

**Reply by Navid200** [reply]
I assume you mean that's the second date on the box.
If yes, it should work.  But, no way to know how long it will work.  It may work for 110 days.

It's not a good idea to have a new transmitter sit in the box.  The battery ages even if you don't use it.

**Reply by GLoren1** [reply]
Thanks.   I updated the transmitter ID with the new transmitter (12/2021 is second date on box).   15 minutes later, status is "not connected".  Should I assume this transmitter's battery is dead?

**Reply by Navid200** [reply]
Where is the new transmitter?
Is it still in the box?
Or, is it in the sensor?

**Reply by GLoren1** [reply]
I took it out of the box and placed it near the phone where xDrip is installed.   Does it need to be in the sensor before started?

**Reply by Navid200** [reply]
Either in the sensor, or you can hold the transmitter in your hand with your thumb on the two contacts.  You will need to hold it like that for a while.
Otherwise, it won't be transmitting.

**Comment by GLoren1** [comment]
Since the sensor never never started correctly,  I didn't think I  needed to remove the transmitter each time. I will try that now though. 

(I now have used the instructions and posted again with my results above)

**Reply by Navid200** [reply]
What do the logs show?
Please note that the logs may contain passwords or serial numbers.  You can blackout such before posting.

**Reply by GLoren1** [reply]
SENSOR
15:38   Sensor stopped at 2022-01-08 15:36:07

OB1

15:38  Stopping Sensor session due to start failure 
15:38  Unusual Session Start Failure
15:38  Session start failed info: 1969-16-31 18:00:00   1969-12-31

15:38  Session Start filed: Clock not synchronized or other error

**Reply by GLoren1** [reply]
Thanks. It's connected now and warming up.   Am I correct that  there's no way to bypass the 2 hour warm up period?

Thanks again

**Reply by Navid200** [reply]
No, there isn't.

Did you wait for the voltages of the new transmitter to populate before starting the sensor?
https://github.com/Navid200/xDrip/wiki/Starting-a-G6-Sensor-or-New-Transmitter

**Reply by GLoren1** [reply]
No, I didn't wait.  Would you recommend waiting for 2 hours, or taking the transmitter out now and starting it?

**Reply by Navid200** [reply]
No, if something is wrong, it should fail soon.
If it doesn't, it should be OK.

Do you have proper connectivity now?
https://github.com/Navid200/xDrip/wiki/Proper-Connectivity

And what is Voltage B?

**Reply by GLoren1** [reply]
It does not have proper connectivity, e.g., last reading was 30 minutes ago and not voltage readings.

**Reply by Navid200** [reply]
Please post a screenshot of the system status page.
You can blackout the serial number.

**Comment by GLoren1** [comment]
![trash6](https://user-images.githubusercontent.com/62320462/148665671-42359a91-f4d0-4395-a6e3-056d54499b79.jpg)

**Reply by Navid200** [reply]
The first thing you need to do is to clear the queue:
https://github.com/Navid200/xDrip/wiki/Clear-Queue

Then, restart the phone.
Monitor the system status page.
Wait 10 minutes.
Do you have proper connectivity now?

If at any point you get a pair request, you need to approve.  You won't if the phone has Android 9 or less.

**Reply by GLoren1** [reply]
I just did that (had already approved pair request).   Now, I still don't have the voltage readings, but the sensor status shows "warming up" and the Last connected time is "2 minutes ago".

**Reply by Navid200** [reply]
Did you clear the queue?
If the connectivity persists, the voltages will be populated shortly.

If you did clear the queue, you can ignore the warm-up.
First things first:  Battery voltages approving reasonable battery and proper connectivity.
Then, and only then, you can worry about starting a sensor.

**Reply by GLoren1** [reply]
Yes, I had cleared the queue.   And, the voltages were populated a couple minutes afterward.   So, I think the problem is fixed. I'll know after the remaining 60 minutes of warm up.  Thanks again.

**Reply by Navid200** [reply]
It's pointless to wait.
You started the sensor when there was no connectivity to the transmitter, which you should not have.
That started a timer in xDrip without really starting the sensor.  How could you have started the sensor without being connected to the transmitter?

**Reply by GLoren1** [reply]
Not sure, but I restarted and it worked; however, the connection seems to get lost every hour or two.  The only way I can regain connection is to change the transmitter ID in the settings, change it back to the real ID and re-pair.  Doing this every hour or two is getting old    My voltage readings are: 

Voltage A 309
Voltage B 293

**Reply by Navid200** [reply]
These are the recommended steps for intermittent connectivity:
https://navid200.github.io/xDrip/docs/Intermittent.html
